Doppler.PushContact/Services/PushContactService.cs

```diff
@@ -265,6 +267,79 @@ public async Task<IEnumerable<string>> GetAllDeviceTokensByDomainAsync(string do
             }
         }
 
+        public async Task<ApiPage<DomainInfo>> GetDomains(int page, int per_page)
+        {
+            try
+            {
+                var domainsFiltered = await PushContacts.Aggregate()
+                    .Group(new BsonDocument
+                    {
+                        { "_id", $"${PushContactDocumentProps.DomainPropName}" },
+                        { "PushContactActiveQuantity",
+                            new BsonDocument("$sum",
+                                new BsonDocument("$cond",
+                                    new BsonArray
+                                    {
+                                        new BsonDocument("$eq",
+                                            new BsonArray
+                                            {
+                                                "$deleted",
+                                                false
+                                            }),
+                                            1,
+                                            0
+                    }))},
+                    { "PushContactInactiveQuantity",
+                        new BsonDocument("$sum",
+                        new BsonDocument("$cond",
+                        new BsonArray
+                    {
+                        new BsonDocument("$eq",
+                        new BsonArray
+                            {
+                                "$deleted",
+                                true
+                            }),
+                        1,
+                        0
+                    })) }
+                    })
+                    .Sort(new BsonDocument("_id", 1))
+                    .Project(new BsonDocument
+                            {
+                                    { "_id", 0 },
+                                    {$"{PushContactDocumentProps.DomainPropName}", "$_id"},
+                                    {"PushContactInactiveQuantity", 1},
+                                    {"PushContactActiveQuantity", 1},
+                            })
+                    .Skip(page)
+                    .Limit(per_page)
+                    .ToListAsync();
+
+                var newPage = page + domainsFiltered.Count;
+
+                var domainList = domainsFiltered
+                    .Select(x =>
+                    {
+                        return new DomainInfo()
+                        {
+                            Name = x.GetValue(PushContactDocumentProps.DomainPropName, null)?.AsString,
+                            PushContactActiveQuantity = x.GetValue("PushContactActiveQuantity", null).ToInt32(),
+                            PushContactInactiveQuantity = x.GetValue("PushContactInactiveQuantity", null).ToInt32()
+                        };
+                    })
+                    .ToList();
```

Comment:
Very minor, but I think that it is less verbose and clear

```suggestion
                var domainList = domainsFiltered
                    .Select(x => new DomainInfo()
                    {
                        Name = x.GetValue(PushContactDocumentProps.DomainPropName, null)?.AsString,
                        PushContactActiveQuantity = x.GetValue("PushContactActiveQuantity", null).ToInt32(),
                        PushContactInactiveQuantity = x.GetValue("PushContactInactiveQuantity", null).ToInt32()
                    })
                    .ToList();
```